holoviews/plotting/bokeh/element.py

```diff
@@ -104,6 +103,12 @@ class ElementPlot(BokehPlot, GenericElementPlot):
           * timeout   - Timeout (in ms) for checking whether interactive
                         tool events are still occurring.""")
 
+    sizing_mode = param.ObjectSelector(default='fixed',
+                                       objects=[
+            'fixed', 'scale_width', 'scale_height',
+            'scale_both', 'stretch_both'
+        ], doc="""Defines how the plot scales when resized.""")
+
```

Comment:
Why not just work on a branch if it is totally WIP? Once you make a PR you make the proposed changes public in a way that invites comment even if the status is WIP.

I would recommend working on a branch and only turning it into a PR when you require some sort of feedback.